release-notes.txt

```diff
@@ -0,0 +1,237 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added support for Spanning Tree Protocol (STP, RSTP and MSTP) on bonded interfaces
```

Comment:
This was already part of 4.4.0 (https://github.com/bisdn/bisdn-linux/blob/v4.4.0/release-notes.txt#L10)

Reply:
Our testing pipelines however were allowed to fail for bonded STP up until two weeks ago (when we no longer saw failing bonded stp tests), should we reword it to say 'now stable support added'?

Reply:
It feels weird having that tied to the list of baseboxd changes, since this isn't a baseboxd change.

I would maybe add a line to the New in this release along side
```
- stp on bond interfaces is now fully supported and considered stable
```

Do we consider it stable? @jklare